atomic_reactor/inner.py

```diff
@@ -343,31 +355,27 @@ def build_docker_image(self):
                 self.autorebuild_canceled = True
                 raise
 
-            start_time = datetime.datetime.now()
-            self.plugins_timestamps['dockerbuild'] = start_time.isoformat()
-
-            build_result = self.builder.build()
-
+            logger.info("running buildstep plugins")
+            buildstep_runner = BuildStepPluginsRunner(self.builder.tasker, self,
+                                                     self.buildstep_plugins_conf,
+                                                     plugin_files=self.plugin_files)
             try:
-                finish_time = datetime.datetime.now()
-                duration = finish_time - start_time
-                seconds = duration.total_seconds()
-                logger.debug("build finished in %ds", seconds)
-                self.plugins_durations['dockerbuild'] = seconds
-            except Exception:
-                logger.exception("failed to save build duration")
-
-            self.build_logs = build_result.logs
-
-            self.build_failed = build_result.is_failed()
+                self.build_result = buildstep_runner.run()
+
+                if self.build_result.is_failed():
+                    self.builder.is_built = False
```

Comment:
Isn't setting to `False` here redundant? Once exception is raised the `except` block below will do the same.